tests/features/collection/collection_overview.feature

```diff
@@ -2,17 +2,13 @@
 Feature: Collections Overview
 
   Scenario: Check visibility of "Collections" menu link.
-    Given I am an anonymous user
-    Then I should see the link "Collections"
-    When I click "Collections"
-    Then I should see the heading "Collections"
-    # Check that all logged in users can see and access the link as well.
+    # Check that all logged in users can see and access the link.
```

Comment:
Hi @idimopoulos,
so should I revert these changes in behat tests? If I revert them, they will be behat errors and I won't be able to push this branch.

I have general question. All changes in behat tests should be processed this way? For example, on the designs I have different login text ("Sign in" instead "Log in") should I also add information about it in the ticket ISAICP-2639? Or can I change it in this Pull request.